pages/common/wasm2wat.md

```diff
@@ -0,0 +1,11 @@
+# wasm2wat
+
+> Convert a file from the WebAssembly binary format to the text format.
```

Comment:
Perhaps `WebAssembly binary format to text format` would read better.
Drop the `the`.
For line 3, 5

Reply:
Not sure on that, @mfrw. The `the` is important in making the sentence read naturally, IMHO.

Reply:
@sbrl I have to improve my english :)

Reply:
Practice makes perfect :-)

Reply:
:)